### ISSUE TITLE
Feature request: Pass-through source subtitles

### ISSUE THREAD

**Issue by cybergrimes** [kmcclive/tricycle#87]
It would be great if we could include source subtitles with the output without using the overlay function. Currently this requires a remux of the Tricycle output and source file after the transcode job has finished.

thanks!

**Comment by kmcclive** [comment]
@cybergrimes Would it meet your needs if I added a setting to the Options/Preferences screen to disable the burn-in/overlay functionality and pass them through instead?  This solution would be a global setting, not per job.  In other words, if you wanted if you wanted passthrough for one job but overlay for the other, you would have to go into the Options/Preferences screen in between jobs.  This also would only allow you to select one subtitle track like it is today.

**Comment by cybergrimes** [comment]
@kmcclive Yes that would meet the need. I never use the burn-in option so disabling at a global level is great for this use case.

ps love Tricycle, perfect for HDR to SDR transcodes without getting caught up in a million settings.
